mod_test/models.py

```diff
@@ -73,8 +72,7 @@ def stages() -> List[Tuple[str, str]]:
         :return: stages available for test
         :rtype: list
         """
-        return [TestStatus.preparation, TestStatus.building,
```

Comment:
Hmm, why don't we have `TestStatus.cancelled` here?

Reply:
Canceled is not one of the stages; tests could be canceled at any point during the test run. Hence we only want to display "Preparation -> Testing -> Completed"  in our frontend.